lib/service/Service.ts

```diff
@@ -158,11 +162,27 @@ class Service {
   }
 
   /**
-   * Demo method to execute a Raiden Token Swap through XUD.
-  */
-  public tokenSwap = async ({ target_address, payload, identifier }: { target_address: string, payload: TokenSwapPayload, identifier: string }) => {
+   * Execute an atomic swap
+   */
+  public executeSwap = async ({ target_address, payload, identifier }: { target_address: string, payload: TokenSwapPayload, identifier: string }) => {
     return this.raidenClient.tokenSwap(target_address, payload, identifier);
   }
+
+  /*
+   * Subscribe to incoming peer orders.
+   */
+  public subscribePeerOrders = async () => {
+    this.pool.on('packet.order', (order: orders.PeerOrder) => this.emit('peerOrder', order));
```

Comment:
I think it makes more sense to implement the subscribes via function-level callbacks, and not via class-level event emitter. So just provide a callback to `subscribePeerOrders`, and call it instead of `this.emit(...`